### PR TITLE
main == preset-commontz

### PR DESCRIPTION
**Pull request by suryavirkapur**
image

### Uncommon
- Use Data `.toString()` so I can get the time according to my current TZ.

### CODE REVIEW

**Comment by cybercoder-naj** [comment]
Thanks for the PR; I will look into this soon.

File: src/presets/index.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { Preset } from '../types';
 import common from './common';
+import uncommon from './common';
```
Comment (cybercoder-naj) [inline src/presets/index.ts:3]: I think you have a typo here, it should import from `./uncommon`

File: src/presets/uncommon.ts
Comment (cybercoder-naj) [inline src/presets/uncommon.ts:1]: I think `uncommon` is a confusing name for this preset. Can you think of something else?

**Review by cybercoder-naj — CHANGES_REQUESTED** [review]
Additionally, it'll be great to have a screenshot image inside `screenshots/` folder so I put it in the wiki.

File: src/presets/uncommon.ts
Comment (suryavirkapur) [reply src/presets/uncommon.ts:1]: I named it `commontz` which suggests it's just the common design with a timezone.

**Comment by suryavirkapur** [comment]
You can review the changes now. 
- sk

**Pull request merged**